### DISCUSSION TITLE
Is it possible to keep static files in any folder rather than `public`?

### DISCUSSION THREAD

**Discussion by lumenwrites** [vercel/next.js#29294] [Help]
I have a markdown blog generated out of `.mdx` files that are kept in the `content` folder. It would be much more convenient if I could keep the images for these posts also in the `content` folder (there's a number of reason for that, for one - I'm using Obsidian to write the blog posts, and to display images inside this tool I need them to be in the same folder).

I have tried making a symlink from the `content/illustrations` folder to `public/illustrations`, but it gives me build errors on Vercel, it says `[Error: ENOENT: no such file or directory, open '/vercel/path1/public/illustrations']`.

What else can I try to accomplish this?

**Comment by PsyGik** [comment]
You'll need to manually update the webpack config to bundle the images using `file-loader` plugin.

> WARNING: untested code

```js
config.module.rules.push({
      test: /\.(svg|png|jpe?g|gif|mp4)$/i,
      use: [
        {
          loader: 'file-loader',
          options: {
            publicPath: '/_next',
            name: 'static/media/[name].[hash].[ext]',
          },
        },
      ],
    });
```
